scripts/chow_ni_tomography/step2_map_chow_onto_ep2020_grid.py

```diff
@@ -0,0 +1,75 @@
+#import utm
```

Comment:
Commented import could be removed? I don't see it being used by any commented code either